scss/forms/_form-control.scss

```diff
@@ -28,6 +29,11 @@
     border: 0;
   }
 
+  // Set the height of date inputs in Safari
+  &::-webkit-date-and-time-value {
+    height: if(unitless($input-line-height), $input-line-height * 1em, $input-line-height);
```

Comment:
I'm not sure why this needed. I remove this and checked on iOS 13 Safari, but I can't see the effect.

Reply:
I thought we could remove the min-height when this is set. But suddenly I rememered https://github.com/twbs/bootstrap/pull/29124, so this is moot.